Prolific/Prolific/View Controllers/Home View Controller/HomeViewController.m

```diff
@@ -59,7 +59,7 @@ - (void)loadProjects {
     DAO *dao = [[DAO alloc] init];
     [dao getAllProjectsWithCompletion:^(NSArray * _Nonnull projects, NSError * _Nonnull error) {
         if (projects) {
-            self.projectArray = projects;
+            self.projectArray = (NSMutableArray *) projects;
```

Comment:
So this is kind of an anti-pattern. You are able to cast this to an nsmutablearray because getAllProjectsWithCompletion is not honoring its contract.

When you're casting it usually means you have a broken or sub-optimal contract. 

1. Ideally the `getAllProjectsWithCompletion` would not return an NSMutableArray 
2. If `getAllProjectsWithCompletion` ever changes internally, your code would break

If you need a mutable array it is better to do `[projects mutableCopy]` so you aren't relying on an implementation detail of `getAllProjectsWithCompletion`